### DISCUSSION TITLE
Could you add one for Linewize and Smoothwall safegaurding client

### DISCUSSION THREAD

**Discussion by Reloadandtryagain** [3kh0/ext-remover#587] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Yeah my admins have 2 blockers on us. I would like something that can bypass this. I used to have Iboss but i passed that myself.(2018) I can't bypass this. Please Help

**Comment by viperbot2000** [comment, marked as answer]
give details about your blocker

**Reply by Reloadandtryagain** [reply]
![image](https://user-images.githubusercontent.com/82478688/228377045-628d8eae-b7f2-4498-8999-adffc4027de7.png)

**Reply by Aka-but-nice** [reply]
DM me on discord, "Aka, but nice#5094"

**Comment by Reloadandtryagain** [comment]
it says i can't here: himself#2176

**Reply by Aka-but-nice** [reply]
Oh, I'll do it

**Reply by Aka-but-nice** [reply]
Alright

**Comment by amelia-bartlett** [comment]
> Yeah my admins have 2 blockers on us. I would like something that can bypass this. I used to have Iboss but i passed that myself.(2018) I can't bypass this. Please Help

Does smoothwall even block anything?  My organisation uses it as well, and I have not noticed any blocking (yet).

You can probably just attack its connection to its backend with pi-hole or something (similar to how you render Securly inoperable)

**Reply by amelia-bartlett** [reply]
Also, please give me an example of a page (preferably something that was accidentally blocked) that is blocked.  AFAIK smoothwall just spies on your activity, it doesn't actively block anything.  (Visigo did, before it was bought up)

Might run it through crxcavator and see if I can find any API calls that are relevant to blocking stuff.

**Reply by amelia-bartlett** [reply]
Haven't found where it blocks stuff, but here is the backend URL, if you want to try and prevent it from communicating
![image](https://user-images.githubusercontent.com/44680463/233165384-b5f85cbe-2802-441a-91c1-edbe4a8cdd0e.png)

**Comment by Reloadandtryagain** [comment]
Sorry i took so long. I can't acess discord anymore.
 and here's the screenshot.
![image](https://user-images.githubusercontent.com/82478688/233446744-84b705d5-0860-49f0-aa66-40d10185b8a8.png)
